### ISSUE TITLE
High contrast theme support

### ISSUE THREAD

**Issue by mcoker** [patternfly/react-catalog-view#87]
Problem statement
Users with specific vision requirements find the interface difficult to read and navigate because the existing theme lacks sufficient color contrast. "As a PatternFly user, I would like to have more accessible options for my vision requirements. (from https://github.com/patternfly/patternfly-design/issues/1357)"

Goal
Update catalog view styles and any usage documentation/suggestions/demos needed to support products' ability to utilize HC catalog view in their UIs.

Links
* https://github.com/patternfly/patternfly-design/issues/1357 - Design epic for HC theme
* https://github.com/patternfly/patternfly-design/issues/1446 - Design epic for catalog view styles
* [Figma design for high contrast](https://www.figma.com/design/iT76DS020Ry1Wswfc0Hmes/branch/wKcOq7IrzfL1gEK2mocd6r/High-Contrast-Exploration?m=auto&node-id=10541-51438&t=RZq1Gz8DNJlvDpOo-1)
